configs/opencv-ovms/scripts/grpc_infer_binary_bit.py

```diff
@@ -0,0 +1,177 @@
+#

```

Comment:
We should look into extracting the common OVMS python code from here and configs/opencv-ovms/scripts/grpc_infer_binary_maskrcnn-omz.py to create a more generic "python inference" in case we add more models in the future.

Reply:
Address in future PR